components/server/src/prebuilds/prebuilt-status-maintainer.ts

```diff
@@ -55,7 +55,7 @@ export class PrebuildStatusMaintainer implements Disposable {
                 this.handlePrebuildFinished(ctx, msg),
             ),
         );
-        this.disposables.push(repeat(this.periodicUpdatableCheck.bind(this), 60 * 1000));
+        this.disposables.push(repeat(this.periodicUpdatableCheck.bind(this), MAX_UPDATABLE_AGE / 2));
```

Comment:
That's fine, in that case maybe a second const which defines the CHECK_UPDATABLE_INTERVAL = MAX_UPDATABLE / 2. 

Up to you, mostly trying to say that changing the const value at use time is a bit of a gotcha. It's generally fine to convert units, but halving is not what I'd expect at use time, but at definition time.

Reply:
There is no need for definition in this case because the value is used just once and is not meant to be configurable directly but is relative to MAX_UPDATABLE. Also the expression is used in a context where it is very clear what it does (even more so than if we were introducing a named constant)